registry/server/appRoutes/routes/updateAppRoute.ts

```diff
@@ -38,14 +38,25 @@ const validateRequestBeforeUpdateAppRoute = validateRequestFactory([
 ]);
 
 const updateAppRoute = async (req: Request<UpdateAppRouteRequestParams>, res: Response) => {
-    await validateRequestBeforeUpdateAppRoute(req, res);
+    try {
+        await validateRequestBeforeUpdateAppRoute(req, res);
+    } catch(err) {
+        res.status(422).send(err);
+        return;
+    }
 
     const {
         slots: appRouteSlots,
         ...appRoute
     } = req.body;
     const appRouteId = req.params.id;
 
+    const countToUpdate = await db('routes').where('id', appRouteId);
```

Comment:
As I see, we have this checking inside of the transaction
Please look at line 63